.gitignore

```diff
@@ -13,6 +13,3 @@
 
 # binaries kind and kubectl from make are in bin
 bin
-# from make create-cluster.  We don't store these.
-go.mod
-go.sum
```

Comment:
I'm not sure we want this... we definitely do not want a run of a test to create a "dirty" env.